MIGRATION.md

```diff
@@ -128,36 +128,39 @@ resource "stackit_logme_credential" "example-credential" {
+| stackit_mongodb_flex_instance            | stackit_mongodbflex_instance            | :white_check_mark: | [project_id],[instance_id] |  |
+| stackit_mongodb_flex_user                | stackit_mongodbflex_user                | :warning: | [project_id],[instance_id],[user_id] | `password` field will be empty |
+| stackit_object_storage_bucket            | stackit_objectstorage_bucket            | :white_check_mark: | [project_id],[name] |  |
+| stackit_object_storage_credential        | stackit_objectstorage_credential        | :white_check_mark: | [project_id],[credentials_group_id],[credential_id] |  |
 | stackit_object_storage_credentials_group | stackit_objectstorage_credentials_group | :white_check_mark: | [project_id],[credentials_group_id] |  |
-| stackit_object_storage_project |  | |  | Resource deprecated |
-| stackit_opensearch_credential | stackit_opensearch_credential | :white_check_mark: | [project_id],[credentials_group_id],[credential_id] |  |
-| stackit_opensearch_instance | stackit_opensearch_instance | :white_check_mark: | [project_id],[instance_id] |  |
-| stackit_postgres_credential | stackit_postgresql_credential | :white_check_mark: | [project_id],[credentials_group_id],[credential_id] |  |
-| stackit_postgres_flex_instance | stackit_postgresflex_instance | :white_check_mark: | [project_id],[instance_id] |  |
-| stackit_postgres_flex_user | stackit_postgresflex_user | :warning: | [project_id],[instance_id],[user_id] | `password` field will be empty |
-| stackit_postgres_instance | stackit_postgresql_instance | :white_check_mark: | [project_id],[instance_id] |  |
-| stackit_project | stackit_resourcemanager_project | :white_check_mark: | [container_id] |  |
-| stackit_rabbitmq_credential | stackit_rabbitmq_credential | :white_check_mark: | [project_id],[credentials_group_id],[credential_id] |  |
-| stackit_rabbitmq_instance | stackit_rabbitmq_instance | :white_check_mark: | [project_id],[instance_id] |  |
-| stackit_redis_credential | stackit_redis_credential | :white_check_mark: | [project_id],[credentials_group_id],[credential_id] |  |
-| stackit_redis_instance | stackit_redis_instance | :white_check_mark: | [project_id],[instance_id] |  |
-| stackit_secrets_manager_instance | stackit_secretsmanager_instance | :white_check_mark: | [project_id],[instance_id] |  |
-| stackit_secrets_manager_user | stackit_secretsmanager_user | :warning: | [project_id],[instance_id],[user_id] | `password` field will be empty |
+| stackit_object_storage_project           |                                         | | | Resource deprecated |
+| stackit_observability_credential         | stackit_observability_credential        | :x: | |  |
+| stackit_observability_instance           | stackit_observability_instance          | :white_check_mark: | [project_id],[instance_id] |  |
+| stackit_observability_job                | stackit_observability_scrapeconfig      | :white_check_mark: | [project_id],[instance_id],[name] |  |
```

Comment:
These are not part of the community provider, so the hint should be added to lines 133-135 (something like "Service deprecated, use stackit_observability_...", and keep the "`id` format" column

Reply:
added back the id infos and extended the notes part as suggested